### ISSUE TITLE
Video Playback in Augmented Reality On Detecting Marker Using A Frames

### ISSUE THREAD

**Issue by anilachary** [jeromeetienne/AR.js#385]
**Do you want to request a *feature* or report a *bug*?**

<script src="https://aframe.io/releases/0.8.0/aframe.min.js"></script>

<script src='aframe_lib/artoolkit.min.js'></script>
<script src='aframe_lib/artoolkit.api.js'></script>

<script src='aframe_lib/svd.js'></script> 
<script src='aframe_lib/posit1.js'></script> 
<script src='aframe_lib/cv.js'></script> 
<script src='aframe_lib/aruco.js'></script> 
<script src='aframe_lib/threex-arucocontext.js'></script> 
<script src='aframe_lib/threex-arucodebug.js'></script>

<script src='aframe_lib/THREE.WebAR.js'></script>

<script src='aframe_lib/signals.min.js'></script>
<script src='aframe_lib/threex-artoolkitprofile.js'></script>
<script src='aframe_lib/threex-artoolkitsource.js'></script>
<script src='aframe_lib/threex-artoolkitcontext.js'></script>
<script src='aframe_lib/threex-arbasecontrols.js'></script>
<script src='aframe_lib/threex-armarkercontrols.js'></script>
<script src='aframe_lib/threex-arsmoothedcontrols.js'></script>
<script src='aframe_lib/threex-hittesting-plane.js'></script>
<script src='aframe_lib/threex-hittesting-tango.js'></script>
<script src='aframe_lib/threex-armarkerhelper.js'></script>
<script src='aframe_lib/arjs-utils.js'></script>
<script src='aframe_lib/arjs-session.js'></script>
<script src='aframe_lib/arjs-anchor.js'></script>
<script src='aframe_lib/arjs-hittesting.js'></script>
<script src='aframe_lib/arjs-tangovideomesh.js'></script>
<script src='aframe_lib/arjs-tangopointcloud.js'></script>
<script src='aframe_lib/arjs-debugui.js'></script>
<script src='aframe_lib/threex-armultimarkerutils.js'></script>
<script src='aframe_lib/threex-armultimarkercontrols.js'></script>
<script src='aframe_lib/threex-armultimarkerlearning.js'></script>

<script src="aframe_lib/system-arjs.js"></script>
<script src="aframe_lib/component-anchor.js"></script>
<script src="aframe_lib/component-hit-testing.js"></script>
<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>

<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">

<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>

<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

<link type="text/css" rel="stylesheet" href="style/style.css">
<script src="js/style.js"></script>
<body style="margin: 0 auto;overflow: hidden;">
  
   <div class="row">
      <div class="col-lg-6">
      </div>
      <div class="col-lg-6 pull-right" >
         <button onclick="skip()" class="btn btnskips" style="z-index: 10000; top:10px;position: fixed; right: 10px">SKIP</button>
	  
		</div>
	  
   </div>
   <div class="timerDisplay" style="position:fixed;top:0px;left:0px; height:40px;font-size:40px;color:white;padding:10px;line-height: 40px;">
		You Will Be Redirected In - <span id="m"></span>
		<span id="l">:</span>
		<span id="s"></span> / 01:03
	 </div>
   
   <div class="row">
      <div class="col-lg-12">
         
         <a-scene embedded arjs='trackingMethod: best;'>
            
            <a-anchor hit-testing-enabled='true'>
               <a-entity>
                  <video  type="video/mp4"  onended="redirect()" id="penguin-sledding" autoplay="true"   loop="false" src="resources/speed.mp4" webkit-playsinline >
                  
               </a-entity>
               
               
               </a-anchor>
           
            
            <a-marker type="pattern" url="https://ladmin.immersionslabs.com/upload/web_lcatalog_resources/pattern1.patt">
                <a-video vidhandler src="#penguin-sledding" position="0 0 0" src="#imge" rotation="-90 0 0"></a-video>
             </a-marker> 
            <a-entity light="color: #ccccff; intensity: 1; type: ambient;" visible=""></a-entity>
            <a-entity camera></a-entity>

         </a-scene>
      </div>
   </div>
   <script>
       AFRAME.registerComponent('vidhandler', {
         init: function () {
          this.toggle = false;
          document.querySelector("#penguin-sledding").pause(); //reference to the video
         },
         tick:function(){  
         if(document.querySelector("a-marker").object3D.visible == true){
           if(!this.toggle){
             this.toggle = true;
             document.querySelector("#penguin-sledding").play();
           }
         }else{
           this.toggle = false;
           document.querySelector("#penguin-sledding").pause();
         }
         }
         });
   </script>
</body>`
). -->

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Operating System and Device Name**
